addons/website/static/tests/tours/snippet_image_quality.js

```diff
@@ -0,0 +1,55 @@
+/** @odoo-module */
+
+import wTourUtils from 'website.tour_utils';
+
+wTourUtils.registerWebsitePreviewTour('website_image_quality', {
+    test: true,
+    url: '/',
+    edition: true,
+}, [
+    wTourUtils.dragNDrop({
+        id: 's_text_image',
+        name: 'Text - Image',
+    }),
+    {
+        content: "Select image",
+        trigger: 'iframe .s_text_image img',
+    },
+    {
+        content: "Set low quality",
+        trigger: 'we-customizeblock-options:has(we-title:contains("Image")) we-range[data-set-quality] input',
+        run: 'range 5',
+    },
+    {
+        content: "Wait for image update: NOT original image",
+        trigger: 'iframe .s_text_image img:not([src$="s_text_image_default_image"])',
+        run: () => {}, // It is a check.
+    },
+    {
+        content: "Check image size",
+        // Reached size cannot be hardcoded because it changes with
+        // different versions of Chrome.
+        trigger: 'we-customizeblock-options:has(we-title:contains("Image")) .o_we_image_weight:contains(" kb"):not(:contains("16.1 kb"))',
+        run() {
+            // Make sure the reached size is smaller than the original one.
+            if (parseFloat(this.$anchor[0].innerText) >= 47.5) {
+                console.error("Image size should be smaller than original");
+            }
+        },
+    },
+    {
+        content: "Set high quality",
+        trigger: 'we-customizeblock-options:has(we-title:contains("Image")) we-range[data-set-quality] input',
+        run: 'range 99',
+    },
+    {
+        content: "Wait for image update: back to original image",
+        trigger: 'iframe .s_text_image img[src$="QUntVvdqEdljipEUu7ez5lb6IqyGKzQmRxVnZX/2Q=="]',
+        run: () => {}, // It is a check.
```

Comment:
I'm not sure it's a good idea to check that random src. And thinking about it... shouldn't it be the original image src here ? Without the current IMP, the image at this step would be bigger than the original? If yes, with your IMP, I would have thought that the src here would be the same as the original.. Am I missing something :thinking: